### PR TITLE
feat: show some useful information in settings screen

### PR DESCRIPTION
**Pull request by bonomat**
I found myself wanting to know what id I have so that I can look it up on the maker side.

Design needs some love, but it's functional:

image

### CODE REVIEW

**Review by da-kami — APPROVED** [review]

**Comment by da-kami** [comment]
This PR supersedes https://github.com/itchysats/10101/pull/499

File: lib/menu.dart [hunk 1]
```diff
@@ -86,9 +87,11 @@ class Menu extends StatelessWidget {
       ListTile(
         title: const Text("Settings"),
         leading: const Icon(Icons.settings),
-        // TODO: Add `selected` when added route (PR is open)
```
Comment (klochowicz) [inline lib/menu.dart:89]: it's missing the `selected` - TODO is removed and the problem not addressed?

**Comment by klochowicz** [comment]
> This PR supersedes #499

499 also added the icon and the behaviour to highlight the current tile, which are missing from this PR. 

also: it's a bit awkward to have "info" in "settings". perhaps they've both have their place?

File: lib/menu.dart (hunk 1 above)
Comment (klochowicz) [reply lib/menu.dart:89]: https://github.com/itchysats/10101/pull/518

**Pull request merged**